assets/app/view/game/hex.rb

```diff
@@ -62,6 +62,19 @@ def render
             points: FRAME_COLOR_POINTS,
           }
           children << h(:polygon, attrs: attrs)
+
+          if (color2 = @tile&.frame&.color2)
+            attrs = {
+              stroke: color2,
+              'stroke-width': FRAME_COLOR_STROKE_WIDTH,
```

Comment:
we do style strkeWidth: now

Reply:
OK. Was just cut and pasting from existing code in the same file. I'll update all the old style attributes in the file.

Reply:
So, strokeWidth: FRAME_COLOR_STROKE_WIDTH doesn't actually work.

Reply:
Not sure SVG styles work the same.